build.gradle

```diff
@@ -191,3 +192,11 @@ task codeCoverageReport(type: JacocoReport) {
     }
 }
 
+sonarqube {
+    properties {
+        property "sonar.projectKey", "aneeshafedo_ballerina-openapi"
+        property "sonar.organization", "aneeshafedo"
```

Comment:
Let's change these to the upstream repo properties, once available

Reply:
Updated